src/ontodia/data/rdf/rdfCacheableStore.ts

```diff
@@ -0,0 +1,294 @@
+                    this.labelsMap[element].push(triple);
+                    this.labelsMap[element].sort((a, b) => {
+                        const index1 = LABEL_URIS.indexOf(a.predicate.nominalValue);
+                        const index2 = LABEL_URIS.indexOf(b.predicate.nominalValue);
+                        if (index1 > index2) {
+                            return 1;
+                        } else if (index1 < index2) {
+                            return -1;
+                        } else {
+                            return 0;
+                        }
+                    });
+                }
+            }
+        }
+
+        const typeInstances = newGraph.match(
+            null,
+            this.prefs.RDF('type'),
+            null,
+        ).toArray();
+        const typeInstMap: Dictionary<string[]> = {};
+        for (const instTriple of typeInstances) {
+            const type = instTriple.object.nominalValue;
+            const inst = instTriple.subject.nominalValue;
+            if (!typeInstMap[type]) {
+                typeInstMap[type] = [];
+            }
+            if (!this.elementTypes[inst]) {
+                this.elementTypes[inst] = [];
+            }
+            if (typeInstMap[type].indexOf(inst) === -1) {
+                typeInstMap[type].push(inst);
+            }
+            this.elementTypes[inst].push(instTriple);
+        }
+        Object.keys(typeInstMap).forEach(key => this.countMap[key] = typeInstMap[key].length);
+
+        return true;
+    }
+
+    private _getLabels(id: string): RDFGraph {
+        return createGraph(this.labelsMap[id]);
+    }
+
+    private getTypes(id: string): RDFGraph {
+        return createGraph(this.elementTypes[id]);
+    }
+
+    private combineGraphs(graphs: RDFGraph[]): RDFGraph {
+        const triples: Triple[] = [];
+        for (const graph of graphs) {
+            for (const triple of graph.toArray()) {
+                triples.push(triple);
+            };
+        };
+        return createGraph(triples);
+    }
+
+    private multipleMatch(statements: MatchStatement[]): RDFGraph {
+        const triples: Triple[] = [];
+        const graphs = Object.keys(this.rdfStorage.graphs).map(id => this.rdfStorage.graphs[id]);
+        for (const graph of graphs) {
+            for (const triple of graph._graph) {
+                for(const statement of statements) {
+                    if (
+                        ((!statement.object) || statement.object === triple.object.nominalValue) &&
+                        ((!statement.predicate) || statement.predicate === triple.predicate.nominalValue) &&
+                        ((!statement.subject) || statement.subject === triple.subject.nominalValue)
+                    ) {
+                        triples.push(triple);
+                        continue;
+                    }
+                };
+            };
+        };
+        return createGraph(triples);
+    }
+}
+
+export default RDFCacheableStore;
```

Comment:
Please use a named export instead of default export. Named exports make better autocomplete support and easier refactoring.